src/factories/workflow-specific/PIM_WorkflowFactory_v1.sol

```diff
@@ -104,10 +129,12 @@ contract PIM_WorkflowFactory_v1 is
             );
         }
 
-        // if renounced workflow flag is set, renounce admin rights over workflow, else transfer admin rights to initial admin
+        // if renounced workflow flag is set factory keeps admin rights over workflow, else transfer admin rights to initial admin
         if (PIMConfig.isRenouncedWorkflow) {
-            _transferWorkflowAdminRights(orchestrator, address(0));
+            // record the deployer as fee recipient eligible to claim buy/sell fees
+            _pimFeeRecipients[fundingManager] = _msgSender();
         } else {
+            // TODO: check if initial admin is the right target
```

Comment:
`TODO`, just FYI.

Reply:
Thanks! I added this because I found it a bit weird to assign the workflow admin role to the issuance token admin. What I did now is to pass a distinct `admin` param to the factory (within the `PIMConfig` struct) that I use for both: token owner and workflow admin rights. I know this doesn't really belong into this PR, but I hope it's fine anyways 👼🏼